grants_tagger/train.py

```diff
@@ -308,6 +337,7 @@ def train_and_evaluate(
     argparser.add_argument('-f', '--from_same_distribution', type=strtobool, default=False, help="whether train and test contain the same examples but differ in other ways, important when loading train and test parts of datasets")
     argparser.add_argument('--threshold', type=float, default=None, help="threhsold to assign a tag")
     argparser.add_argument('--y_batch_size', type=int, default=None, help="batch size for Y in cases where Y large. defaults to None i.e. no batching of Y")
+    argparser.add_argument('--x_format', type=str, default="List", help="format that will be used when loading the data. One of JSONL,DataFrame")
```

Comment:
you say 'One of JSONL,DataFrame' but the default type is List?

Reply:
oups, good spot, will fix

Reply:
2ca0c15